ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -45,15 +45,15 @@ struct InstanceData
 {
 	mat4x3 tform;
 	vec3 normalMatrixRow0;
-	uint bsdf_instrOffset;
+	uint instr_offset;
 	vec3 normalMatrixRow1;
-	uint bsdf_instrCount;
+	uint rem_pdf_count;
 	vec3 normalMatrixRow2;
 	uint _padding;//not needed
-	uvec2 prefetch_instrStream;
-	uvec2 nprecomp_instrStream;
-	uvec2 genchoice_instrStream;
 	uvec2 emissive;
+	uint prefetch_count;
+	uint nprecomp_count;
+	uint genchoice_count;
```

Comment:
if you put these in front of emissive, you could get rid of the padding, and have everything fit in 8 uvec4s